custom/icds_reports/ucr/data_sources/child_cases_monthly_v2.json

```diff
@@ -1870,7 +1879,21 @@
         "is_nullable": true,
         "type": "expression",
         "column_id": "is_cf"
-      }
+      },
+      {  
+        "display_name": null,
+        "datatype": "string",
+        "type": "expression",
+        "is_primary_key": false,
+        "transform": {},
+        "is_nullable": true,
+        "expression": {
+          "type": "named",
+          "name": "zscore_grading_wfa"
+        },
+        "column_id": "zscore_grading_wfa_display",
```

Comment:
this is a weird name for the column, it is probably nicer to just call it `zscore_grading_wfa`, since its not actually displayed and is more concise the other way.

Reply:
@calellowitz , can the name of a named_expression and configured_indicator be the same? the _display is to differentiate it from the named_expression.